public/index.html

```diff
@@ -10,12 +10,14 @@
     <meta name="mobile-web-app-capable" content="yes">
     <title>Chat with SDSLabs</title>
 
-    <link rel="icon" sizes="302x302" href="https://i.imgur.com/WNRjxyN.jpg">
+    <link rel="shortcut icon" sizes="302x302" href="jinora.ico" >
 
+    
     <script src="bower_components/webcomponentsjs/webcomponents.min.js"></script>
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/mustache.js/0.8.1/mustache.min.js"></script>
     <script src="/socket.io/socket.io.js"></script>
+    <script src="bower_components/favico.js/favico-0.3.7.min.js"></script> 
```

Comment:
You need to update bower.json with the packages that you add, otherwise I can't install them